recipes/toil/meta.yaml

```
@@ -0,0 +1,126 @@
+# copied from https://github.com/bioconda/bioconda-recipes/blob/9c08a16bac9b7d0a076578823d5008b8bbfc466e/recipes/toil/meta.yaml
+{% set name = "toil" %}
+{% set version = "7.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c3f4e9567754f5ad6fe56b90913b7c1c9aaade32b545502a483cd57f70f24777
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --no-cache-dir -vvv
+  entry_points:
+    - toil = toil.utils.toilMain:main
+    - _toil_worker = toil.worker:main
+    - toil-cwl-runner = toil.cwl.cwltoil:main
+    - toil-wdl-runner = toil.wdl.wdltoil:main
+    - toil-wes-cwl-runner = toil.server.cli.wes_cwl_runner:main
+    - _toil_mesos_executor = toil.batchSystems.mesos.executor:main
+    - _toil_contained_executor = toil.batchSystems.contained_executor:executor
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools >=42
+    - wheel
+  run:
+    - python >={{ python_min }}
+    - dill >=0.3.2,<0.4
+    - requests >2,<=2.31.0
+    - docker-py >=3.7.2,<8
+    - urllib3 >=1.26.0,<3
+    - python-dateutil
+    - psutil >=3.0.1,<6
+    - pypubsub >=4.0.3,<5
+    - addict >=2.2.1,<2.5
+    - pytz >=2012
+    - enlighten >=1.5.2,<2
+    - configargparse >=1.7,<2
+    - typing-extensions >=4.6.2,<5
+    - pyyaml >=6,<7
```

```suggestion
  run:
    - python >={{ python_min }}
    - dill >=0.3.2,<0.4
    - requests >2,<=2.31.0
    - docker-py >=6.1.0,<8
    - urllib3 >=1.26.0,<3
    - python-dateutil
    - psutil >=3.0.1,<6
    - pypubsub >=4.0.3,<5
    - addict >=2.2.1,<2.5
    - enlighten >=1.5.2,<2
    - configargparse >=1.7,<2
    - ruamel.yaml >=0.15
    - typing-extensions >=4.6.2,<5
    - coloredlogs >=15,<16
```

Synchronize with https://github.com/DataBiosphere/toil/blob/releases/7.0.0/requirements.txt